Loop54.Shared/Model/Request/GetIndexedAttributeValuesRequest.cs

```diff
@@ -0,0 +1,20 @@
+namespace Loop54.Model.Request
+{
+    /// <summary>Request to the getIndexedAttributeValues API method of the Loop54 e-commerce search engine.</summary>
+    public class GetIndexedAttributeValuesRequest : Request
+    {
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="attributeName">The indexed attribute for which to fetch unique values. Can at maximum be 200 chars long.</param>
```

Comment:
Hmm, I thought we did. Maybe we should, at least for the ones we intend the user to ask about in the API. Don't we limit the GEBA attribute name?

Reply:
Apparently we don't, but it was limited in practice, since we created Entities out of all the categories and there is a max length on Entity id. So I guess now that we don't use Entities for this anymore we no longer have a limit. I'll remove the comment.